packages/components/src/components/ThemeProvider/index.tsx

```diff
@@ -80,3 +81,7 @@ export const ThemeProvider = ({ theme, children }) => {
     </>
   );
 };
+
+ThemeProvider.defaultProps = {
+  theme: codesandboxBlack,
```

Comment:
What's the point of setting a `defaultProp` if there aren't any props passed? 🤔

Reply:
What do you mean? You pass the theme, we used to have to pass it always and now with this you only need to pass it when it changes and not in the static pages

Reply:
@jyash97 You can just use it in the function arguments btw 🙂

```tsx
export const ThemeProvider = ({ children, theme = codesandboxBlack }) => {
  // ...
};
```

Reply:
@SaraVieira But passing the `theme` prop is removed everywhere in this PR, no? 🤔

Reply:
Noo, just in those pages, we also pass it in places in the editor where it changes based on the user theme choice

Reply:
My bad, I thought it was removed everywhere. 😓

Since we're using function components, we can just pass the default prop like we do in functions instead of defining `ThemeProvider.defaultProps`, so I would change that one tho

Reply:
Done @MichaelDeBoey  👋